sdk/go/common/workspace/plugins.go

```diff
@@ -460,7 +473,11 @@ func (source *githubSource) getHTTPResponse(
 
 	addAuth := ""
 	if source.token == "" {
-		addAuth = " You can set GITHUB_TOKEN to make an authenticated request with a higher rate limit."
+		if source.tokenDisabled {
+			addAuth = " You can set GITHUB_TOKEN to a different token to make a request with a higher rate limit."
```

Comment:
```suggestion
			addAuth = " Your current GITHUB_TOKEN doesn't allow access to the repository, so we disabled it for this request.  You can set GITHUB_TOKEN to a different token to make a request with a higher rate limit."
```

Not sure if it's worth giving some more details.

Reply:
Why not